static/app/views/performance/newTraceDetails/index.tsx

```diff
@@ -192,7 +201,7 @@ export function TraceView() {
   }, [queryParams, traceSlug]);
 
   const trace = useTrace();
-  const meta = useTraceMeta();
+  const meta = useTraceMeta([traceSlug]);
```

Comment:
Why do we need an array if there is only one trace slug?

Reply:
A replay has multiple traceIds, so I made useTraceMeta able to handle multiple traceIds.